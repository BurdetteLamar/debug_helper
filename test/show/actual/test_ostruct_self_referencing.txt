---
OpenStruct (message='test_ostruct_self_referencing'):
  Member 0:
    Name:
      Symbol:
        Symbol#to_s: a
        Symbol#size: 1
        Symbol#encoding: !ruby/encoding US-ASCII
    Value: 'OpenStruct #<OpenStruct a=#<OpenStruct ...>>'
