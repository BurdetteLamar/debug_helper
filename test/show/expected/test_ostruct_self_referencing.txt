---
OpenStruct (message='test_ostruct_self_referencing'):
  size: 
  Member 0:
    Name:
      Symbol:
        to_s: a
        size: 1
        encoding: !ruby/encoding US-ASCII
    Value: 'OpenStruct #<OpenStruct a=#<OpenStruct ...>>'
